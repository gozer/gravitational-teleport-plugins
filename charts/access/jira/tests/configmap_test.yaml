suite: Test deployment
templates:
  - configmap.yaml
tests:
  - it: should match the snapshot (smtp on)
    set:
      teleport:
        address: teleport.example.com:1234
      jira:
        url: https://jira.example.com
        username: user@example.com
        apiToken: examplejiratoken
        project: ACC
        Task: Bug
      http:
        listenAddress: ":1234"
        publicAddress: "jira-plugin.example.com"
        keyFile: "server.key"
        certFile: "server.crt"
        verifyClientCert: true

        basicAuth:
          user: "testuser"
          password: "testpassword"
      log:
        output: /var/log/teleport-jira.log
        severity: DEBUG
    asserts:
      - matchSnapshot: {}
