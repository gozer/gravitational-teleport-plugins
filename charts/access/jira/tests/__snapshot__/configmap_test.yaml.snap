should match the snapshot (smtp on):
  1: |
    apiVersion: v1
    data:
      teleport-jira.toml: |
        [teleport]
        addr = "teleport.example.com:1234"
        identity = "/var/lib/teleport/plugins/jira/auth_id"

        [jira]
        url = "https://jira.example.com"
        username = "user@example.com"
        api_token = "examplejiratoken"
        project = "ACC"
        issue_type = "Task"

        [http]
        listen_addr = ":1234"
        public_addr = "jira-plugin.example.com"
        https_key_file = "server.key"
        https_cert_file = "server.crt"



        [https.tls]
        verify_client_cert = true

        [log]
        output = "/var/log/teleport-jira.log"
        severity = "DEBUG"
    kind: ConfigMap
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: teleport-plugin-jira
        app.kubernetes.io/version: 9.0.4
        helm.sh/chart: teleport-plugin-jira-1.0.0
      name: RELEASE-NAME-teleport-plugin-jira
